src/css/components/modal.less

```diff
@@ -24,15 +34,15 @@
   word-wrap: break-word;
 
   .row.full-bleed {
-    margin-left: -20px;
-    margin-right: -20px;
+    margin-left: -@modal-body-padding;
```

Comment:
Nope, unless you're sure that we can overwrite `margin-top` and `margin-bottom` without any side effects.